web/scripts/frame.js

```diff
@@ -83,10 +83,15 @@ messageHandler = function (e) {
         //
         // See also:
         // https://github.com/flutter/engine/blob/master/lib/web_ui/lib/src/engine/configuration.dart
-        if (obj.canvasKitBaseUrl) {
+        if (obj.canvasKitBaseUrl && !obj.useLegacyCanvasKit) {
             window.flutterConfiguration = {
                 canvasKitBaseUrl: obj.canvasKitBaseUrl
             };
+        } else {
+            // Use legacy CanvasKit URL
+            window.flutterConfiguration = {
+                canvasKitBaseUrl: "https://unpkg.com/canvaskit-wasm@0.37.1/bin/"
```

Comment:
But we still support Flutter 3.3, right?
Not sure it'll work with this old one...

Reply:
Yeah the old channel is technically using an incompatible version. It's working for me at least...

Reply:
then we're fine!

Reply:
The stable channel will be the old channel soon